tests/bulk_create/tests.py

```diff
@@ -39,6 +40,13 @@ def test_efficiency(self):
         with self.assertNumQueries(1):
             Country.objects.bulk_create(self.data)
 
+    @skipUnlessDBFeature('has_bulk_insert')
+    @skipUnless(connection.vendor == 'oracle', "Test only for Oracle")
+    def test_oracle_long_text_edge_case(self):
+        """Ticket 22144."""
```

Comment:
It's helpful to describe the issue with a few words, e.g. "Inserting non-ASCII values longer than X characters (#22144)." (I'm not certain that's a correct description of the issue).